src/Security/SecurityConfigUpdater.php

```diff
@@ -80,6 +80,16 @@ public function updateForAuthenticator(string $yamlSource, string $firewallName,
             $firewall['guard']['entry_point'] = $chosenEntryPoint ?? current($firewall['guard']['authenticators']);
         }
 
+        if (!isset($firewall['logout']) && $logoutSetup) {
+            $firewall['logout'] = ['path' => 'app_logout'];
+            $firewall['logout'][1] = $this->manipulator->createCommentLine(
```

Comment:
Just use `$firewall['logout'][]` here and below - it'll append without needing the index.